orchestrator-salt/src/main/resources/salt/salt/cloudera/csd/csd-downloader.sh

```diff
@@ -6,11 +6,22 @@ mkdir -p /opt/cloudera/csd
 cd /opt/cloudera/csd
 
 {% if salt['pillar.get']('cloudera-manager:csd-urls') %}
-{% for url in salt['pillar.get']('cloudera-manager:csd-urls') %}
-curl -L -O -R {{ url }}
-{% endfor %}
+csdUrls=({%- for url in salt['pillar.get']('cloudera-manager:csd-urls') -%}
+{{ url + " " }}
+{%- endfor %})
+
+for url in ${csdUrls[@]}
+do
+  fileName=$(basename $url)
```

Comment:
I would add a log here with url and filename to see what's going on. also I would add a datetime to logs. it usually helps during investigation to see what happened when

Reply:
might use `tee` to have the logs in minion log and the `csd_downloaded` too